packages/mobile/src/fiatExchanges/MoonPayScreen.tsx

```diff
@@ -60,13 +60,14 @@ function MoonPayScreen({ route }: Props) {
   }, [])
 
   return (
-    <View style={styles.container}>
-      {uri === '' ? (
-        <ActivityIndicator size="large" color={colors.greenBrand} />
-      ) : (
-        <WebView source={{ uri }} />
+    <>
+      {uri === '' && (
```

Comment:
I found the condition nicer the way it was before with the ternary, with the current change we run the risk of missing a case at some point and ending up with a blank screen. It's unlikely to happen but 🤷‍♂️